src/test/java/org/cloudburstmc/server/config/CloudburstYamlMappingTest.java

```diff
@@ -141,7 +141,7 @@ void parsingWorks() {
                 yml.getAliases()
         );
 
-        HashMap<String, ServerConfig.World> worldConfig = new HashMap<>();
+      /*HashMap<String, ServerConfig.World> worldConfig = new HashMap<>(); With the new system the old test for seeds is no longer needed
```

Comment:
This tests more than just the seed, and should probably be left in.

Reply:
Also if you do come across any code that isn’t needed anymore, you should probably just remove it or modify it instead of commenting it out